main.css

```diff
@@ -1,16 +1,22 @@
 @import url("https://orangeui.netlify.app/orange-ui.css");
+body{
+    background-color: white;
+}
 .search-bar2{
     margin-top: -10px;
 }
 .navigation-bar{
     /* border: none; */
     height: 60px;
-    width: 100%;
+    /* width: 100%; */
 }
 .btn-login{
     height: 30px;
     margin-top: 20px;
 }
+#search-bar{
+    height: 30px;
```

Comment:
Always use rem avoid using px ...rem is also good for responsive use px upto 10px only and also avoid using id to select element always use classes ..